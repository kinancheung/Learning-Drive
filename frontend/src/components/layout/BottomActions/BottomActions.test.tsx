import { render } from '@testing-library/react';

import { BottomActions } from '.';

describe('BottomActions', () => {
  it('renders without error', () => {
    const view = render(<BottomActions />);

    expect(view.asFragment()).toMatchSnapshot();
  });
});
