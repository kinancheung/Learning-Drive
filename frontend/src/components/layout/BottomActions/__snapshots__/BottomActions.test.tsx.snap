// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomActions renders without error 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-1himdu3"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthMd MuiContainer-disableGutters css-blt0bx-MuiContainer-root"
    >
      <div
        class="css-2ht33p-MuiStack-root"
      />
    </div>
  </div>
</DocumentFragment>
`;
